lms/templates/courseware/program_marketing.html

```diff
@@ -125,15 +125,35 @@ <h1 class="banner-title">${title}</h1>
             </span>
           </li>
           <li class="item">
-            <span>
+            <span class="description">
               <span class="fa fa-tag fa-lg" aria-hidden="true"></span>
               ${_('Price (USD)')}
             </span>
-            <span>
-              ${Text(_('${full_program_price} for entire program')).format(
-                full_program_price=full_program_price
-              )}
-            </span>
+            % if program.get('discount_data') and program['discount_data']['is_discounted']:
+              <span class="price">
+                <span role="group" aria-label="${_('Original Price')}" class="original-price">
+                  ${Text(_('${oldPrice}')).format(
+                    oldPrice=full_program_price_format.format(program['discount_data']['total_incl_tax_excl_discounts'])
+                  )}
+                </span>
+                <span role="group" aria-label="${_('Discounted Price')}" class="discount green-highlight">
+                  ${Text(_('${newPrice}{htmlEnd} for entire program')).format(
+                    newPrice=full_program_price,
+                    htmlEnd=HTML('</span>')
+                  )}
+                <span class="savings green-highlight">
+                  ${Text(_('You save ${discount_value} USD')).format(
```

Comment:
You should be getting the currency from the calculate endpoint. Perhaps use that variable here instead of hard coding `USD`.